common/sensirion_config.h

```diff
@@ -84,13 +102,4 @@
 
 #endif /* __cplusplus */
 
-/**
- * The clock period of the i2c bus in microseconds. Increase this, if your GPIO
- * ports cannot support a 200 kHz output rate. (2 * 1 / 10usec == 200Khz)
- *
- * This is only relevant for the sw-i2c HAL (bit-banging on GPIO pins). The
- * pulse length is half the clock period, the number should thus be even.
- */
-#define SENSIRION_I2C_CLOCK_PERIOD_USEC 10
-
-#endif /* SENSIRION_ARCH_CONFIG_H */
+#endif /* SENSIRION_CONFIGURATION_H */
```

Comment:
```suggestion
#endif /* SENSIRION_CONFIGURATION_H */

```

newline at end of file